dbt/adapters/athena/impl.py

```diff
@@ -213,7 +213,7 @@ def generate_s3_location(
         table_prefix = self._s3_table_prefix(s3_data_dir)
 
         mapping = {
-            S3DataNaming.UUID: path.join(table_prefix, str(uuid4())),
+            S3DataNaming.UNIQUE: path.join(table_prefix, str(uuid4())),
```

Comment:
we need to change the docs in the README and in the new doc page to propagate those changes.